File: javascript/zadanie4.js
var fs = require('fs')

var file = fs.readFileSync('dane.txt').toString('utf8')
var dates = file.split('\r\n')

if (fs.existsSync('wynik.txt'))
    fs.unlinkSync('wynik.txt')

dates.forEach(date => {
    if (String(date) === "Tuesday")
        fs.appendFileSync('wynik.txt', date + '\r\n')
});

File: cpp/zadanie4.cpp
#include <iostream>
#include <fstream>
#include <string>

int main() {
	std::fstream inFile;
	std::fstream outFile;

	inFile.open("dane.txt", std::ios::in);
	outFile.open("wynik.txt", std::ios::out);
	
	std::string date;

	while (!inFile.eof()) {
		date = "";
		inFile >> date;
		if (date == "Tuesday")
			outFile << date << std::endl; 
	}

	inFile.close();
	outFile.close();

	return 0;
}